### PR TITLE
Run limit results/screenshots/logs job less often

### PR DESCRIPTION
**Pull request by Martchus**
see https://github.com/os-autoinst/openQA/pull/2710#issuecomment-580181673

### CODE REVIEW

File: systemd/openqa-enqueue-asset-cleanup.service [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Enqueues an asset cleanup cleanup task for the openQA.
```
Comment (kalikiana) [inline systemd/openqa-enqueue-asset-cleanup.service]: Please don't copy the Czech "the" here again.

Comment (kalikiana) [reply systemd/openqa-enqueue-asset-cleanup.service]: Also, one cleanup is enough ;-)

Comment (Martchus) [reply systemd/openqa-enqueue-asset-cleanup.service]: fixed

Comment (coolo) [reply systemd/openqa-enqueue-asset-cleanup.service]: there is no such thing as 'Czech "the"' - a proper classification of this construct is 'czech overcompensation of missing articles' :)

Comment (Martchus) [reply systemd/openqa-enqueue-asset-cleanup.service]: Note that it has been copied from "the openQA database" and then only "database" has been removed.

**Comment by Martchus** [comment]
The failing CI check is unrelated. I'm working on a fix on another PR.

**Comment by okurz** [comment]
please add more details to the git commit message to give a bit more reasoning. Could be very helpful in some months from now to understand what we were thinking :)

**Comment by Martchus** [comment]
Any thoughts on the failing OBS check?

```
[   91s] error: Failed build dependencies:
[   91s] 	python3-future is needed by openQA-4.6.1580407293.37e88eeff-4.1.noarch
[   91s] 	python3-requests is needed by openQA-4.6.1580407293.37e88eeff-4.1.noarch
```

It looks unrelated but since I've changed the spec file it would actually be great to see if it works.

**Pull request merged**